UnityProject/Assets/Scripts/Clothing/BackPack/InteractableStorage.cs

```diff
@@ -143,6 +143,12 @@ public bool WillInteract(InventoryApply interaction, NetworkSide side)
 		// we need to be the target - something is put inside us
 		if (interaction.TargetObject != gameObject) return false;
 		if (DefaultWillInteract.Default(interaction, side) == false) return false;
+		if (itemStorage.GetItemSlots().All(slot => slot.IsOccupied) && interaction.UsedObject != null)
```

Comment:
what does this linq sorcery do? Chek if them all are occupied?

Reply:
Yes

Reply:
```suggestion
		if (itemStorage.GetNextFreeIndexedSlot() == null && interaction.UsedObject != null)
```

Reply:
Nice, less Linq stuff the better.